src/app/utils/highlights.ts

```diff
@@ -1,3 +1,12 @@
+import { Match, MatchHighlight } from '../../types';
+
+const { VITE_PORT } = import.meta.env;
+const baseUrl = `http://localhost:${VITE_PORT}/api`;
```

Comment:
The server will be deployed with a different hostname, thus, I would add an environment variable for the whole url like `API_URL=https://localhost:3001`.